apps/client/src/main.tsx

```diff
@@ -14,7 +14,16 @@ const root = ReactDOM.createRoot(
 );
 root.render(
   <StrictMode>
-    <StiggProvider apiKey={config.stiggApiKey}>
+    <StiggProvider
+      apiKey={config.stiggApiKey}
+      theme={{
+        layout: {
+          planMinWidth: '330px',
+          descriptionMinHeight: '60px',
+          switchBottomSpacing: '20px',
+        },
+      }}
```

Comment:
I really want to move this theming options to the `StiggPaywall` component but we can do that later